management/migrations/0003_mysql_cache.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        # Add a dependency in here on an existing migration in the app you
```

Comment:
You could probably remove this comment, since it's referring to the `django_mysql` example and might be confusing to people reviewing this. Any rationale to selecting this migration as a dependency?

Reply:
Just to keep order I guess. Probably doesn't even need a dependency though, I could remove it entirely.

Reply:
I think I'll remove the comment but leave the dependency in there just for ordering context incase anything comes after this. Otherwise it's possible something afterward may require this and then have to have two dependencies which can get confusing to me.